glusterd2/volume/store-utils.go

```diff
@@ -5,10 +5,9 @@ import (
 	"encoding/json"
 	"errors"
 
+	"github.com/coreos/etcd/clientv3"
```

Comment:
Include this in end. Order to follow is

- standard imports
- our imports
- third party libs import

Reply:
Resolved.